private.tf

```
@@ -4,7 +4,7 @@ module "private_label" {
   stage      = "${var.stage}"
   name       = "${var.name}"
   delimiter  = "${var.delimiter}"
-  attributes = ["private"]
+  attributes = "${concat(var.attributes,list("private"))}"
```

```suggestion
  attributes = ["${concat(var.attributes,list("private"))}"]
```

Does it work without explicitly specifying list?
In some cases TF complains about list datatype.

---

Yes, it works. The output of `concat` is a list, so it does not need to be in brackets. I tested both with the default and with a list.